lib/mongoose_push.ex

```diff
@@ -13,6 +13,8 @@ defmodule MongoosePush do
   require Logger
   alias MongoosePush.Service
 
+  @behaviour MongoosePush.Notification
```

Comment:
Can you elaborate on why we need this behaviour? As far as I understand it is only used for mocks in tests, is there any other reason?